bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java

```diff
@@ -85,6 +91,18 @@ public void createControl(Composite parent)
 
 	}
 
+	@Override
+	public void dispose()
+	{
+		super.dispose();
+		IWorkbenchPage page = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
+		IViewPart viewPart = page.findView("org.eclipse.ui.navigator.ProjectExplorer");
+		ISelectionProvider selProvider = viewPart.getSite().getSelectionProvider();
```

Comment:
See if you can make use of any method from EclipseHandler, also keep null checks for viewpart

Reply:
@kolipakakondal found only this. 
https://github.com/espressif/idf-eclipse-plugin/blob/b12bd525186fac1a1603df0b7a6071b6169ec5de/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java#L100

null pointer check was also added:
https://github.com/espressif/idf-eclipse-plugin/blob/b12bd525186fac1a1603df0b7a6071b6169ec5de/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java#L102